### PR TITLE
Updated some links in the "How to best contact us" article

### PR DESCRIPTION
**Pull request by ghost**
Removed Forge mention and replaced it with Github
Update links to Github: report a bug and create a PR
Updated links to Addons: create modules and report an issue
Updated link to the Forum

### CODE REVIEW

**Comment by ghost** [comment]
Hope text PRs are taken in account for Hacktoberfest ;)

**Review by PierreRambaud — APPROVED** [review]

File: news/_posts/2017-03-31-how-to-best-contact-us.md [hunk 1]
```diff
@@ -151,13 +151,13 @@ No software is perfect, and PrestaShop is no exception. Our team spends its whol
...
+Please use [this reference guide](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/) to better understand what makes a great Forge ticket.
 
 Then, create your report for the right context. The Core team will review your report and answer back with any questions and /or ideas.
 
-* [PrestaShop 1.7](http://forge.prestashop.com/secure/CreateIssue%21default.jspa?selectedProjectId=11322&issuetype=1) 
-* [PrestaShop 1.6](http://forge.prestashop.com/secure/CreateIssue%21default.jspa?selectedProjectId=10110&issuetype=1) 
-* [Partner or Native Modules](http://forge.prestashop.com/secure/CreateIssue%21default.jspa?selectedProjectId=10080&issuetype=1) 
+* [PrestaShop 1.7](http://forge.prestashop.com/secure/CreateIssue%21default.jspa?selectedProjectId=11322&issuetype=1)
+* [PrestaShop 1.6](http://forge.prestashop.com/secure/CreateIssue%21default.jspa?selectedProjectId=10110&issuetype=1)
+* [Partner or Native Modules](http://forge.prestashop.com/secure/CreateIssue%21default.jspa?selectedProjectId=10080&issuetype=1)
```
Comment (ttoine) [inline news/_posts/2017-03-31-how-to-best-contact-us.md]:
```suggestion
* [PrestaShop 1.7 and modules](https://github.com/PrestaShop/PrestaShop/issues)
* [PrestaShop 1.6 (now end of life)](https://github.com/PrestaShop/PrestaShop-1.6/issues)
```

File: news/_posts/2017-03-31-how-to-best-contact-us.md [hunk 2]
```diff
@@ -216,32 +216,32 @@ Core contributors are expected to know the PrestaShop technologies and tools in
 
 Got everything set up? Find the branch you want to work with:
 
-* [Current development branch](https://github.com/PrestaShop/PrestaShop/tree/develop) (for the next 1.7 minor version). 
+* [Current development branch](https://github.com/PrestaShop/PrestaShop/tree/develop) (for the next 1.7 minor version).
 * [Current 1.6.1.x branch](https://github.com/PrestaShop/PrestaShop/tree/1.6.1.x).
```
Comment (ttoine) [inline news/_posts/2017-03-31-how-to-best-contact-us.md]:
```suggestion
* [Current 1.6.1.x branch](https://github.com/PrestaShop/PrestaShop-1.6).
```

**Review by ttoine — COMMENTED** [review]
let's remove links to the forge.

**Review by ttoine — APPROVED** [review]

**Comment by ttoine** [comment]
should we merge this one ?

**Comment by ghost** [comment]
Good for me!

**Pull request merged**